sbncode/CAFMaker/FillReco.cxx

```diff
@@ -892,6 +892,53 @@ namespace caf
     }
   }
 
+  void FillPlaneLikePID(const anab::ParticleID &particle_id, caf::SRTrkLikePID &srlikepid) {
+
+    // Assign dummy values.
+
+    srlikepid.lambda_muon = 0.;
+    srlikepid.lambda_pion = 0.;
+    srlikepid.lambda_proton = 0.;
+    srlikepid.pid_ndof = 0;
```

Comment:
Why do we change from the previous defaults here?

The object fills the defaults as -5